hyperspace/cosmos/src/light_client.rs

```diff
@@ -0,0 +1,101 @@
+//! This section mainly has been ported from `InformalSystems/hermes/relayer/src/light_client`
+use crate::error::Error;
+use ibc::Height;
+use ics07_tendermint::{client_state::ClientState, ProdVerifier};
+use pallet_ibc::light_clients::HostFunctionsManager;
+use tendermint::trust_threshold::TrustThresholdFraction;
+use tendermint_light_client::{
+	components::{
+		self,
+		io::{AtHeight, ProdIo},
+	},
+	light_client::LightClient as TmLightClient,
+	state::State as LightClientState,
+	store::{memory::MemoryStore, LightStore},
+};
+use tendermint_light_client_verifier::{
+	options::Options as TmOptions,
+	types::{Height as TMHeight, LightBlock, PeerId, Status},
+	PredicateVerifier,
+};
+use tendermint_rpc::{Client, HttpClient, Url};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Verified<H> {
```

Comment:
what does verified is? can you add some docs on this struct?

Reply:
It was copied from `hermes` and actually not used anywhere, so I'll just remove it